ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -39,7 +39,7 @@ SettingsContentBase {
 
     Item {
         id: appearanceContainer
-        anchors.left: parent.left
+        anchors.left: !!parent ? parent.left : undefined
```

Comment:
missing parent? In which case is this happening?

Reply:
that's very usual in StackLayout, in the moment when layouts are switched